src/main/resources/application-test.properties

```diff
@@ -0,0 +1 @@
+spring.autoconfigure.exclude=org.springframework.boot.autoconfigure.jdbc.DataSourceAutoConfiguration, org.springframework.boot.autoconfigure.orm.jpa.HibernateJpaAutoConfiguration
```

Comment:
Configuration to stop automatic database connection when running with test profile